colossus/src/main/scala/colossus/protocols/http/HttpBody.scala

```diff
@@ -5,7 +5,9 @@ import akka.util.ByteString
 import scala.util.Try
 import core.DataBlock
 
-class HttpBody(private val body: Array[Byte], val contentType : Option[HttpHeader] = None)  {
+class HttpBody(private val body: Array[Byte], contentTypeString: Option[String] = None) {
```

Comment:
Decided to be a little bold here and change the constructor, since we probably don't want to give the caller the freedom to pass in non-`ContentType` `HttpHeader`s.